package.json

```diff
@@ -84,7 +88,7 @@
     "@storybook/addon-storysource": "^6.5.7",
     "@storybook/addons": "^6.5.7",
     "@storybook/builder-webpack5": "^6.5.7",
-    "@storybook/ember": "^6.5.7",
+    "@storybook/ember": "6.5.14",
```

Comment:
Storybook fails to build when we install a newer version, I haven't looked into it yet.